gen/blog/import.hoon

```diff
@@ -0,0 +1,20 @@
+:-  %say
+|=  [[now=@da @ our=@p ^] *]
+=/  =path  /(scot %p our)/blog/(scot %da now)/import
+:-  %blog-action
+:-  %import
+|-
+=+  .^(=arch %cy path)
+:: If there is no directory below this, read the file's contents
+?:  (test dir.arch ~)
+  =/  mark
+  ?+  (rear path)  !!
```

Comment:
=/ is an exception to the rule that you want it to be on the same line that it started - you want to indent everything by one here